source/documentation/guides/joiners_movers_leavers.md.erb

```diff
@@ -0,0 +1,43 @@
+---
+title: Joiners Movers Leavers
+weight: 10
+last_reviewed_on: 2024-01-31
+review_in: 12 weeks
+---
+# <%= current_page.data.title %>
+
+The majority of our Joiners/Movers/Leavers processes can be carried out by using the opg-org-infra Terraform to assign permissions. Some tools like Jira or product-specific admin tools may require additional work. In general the JML process will be a combination of Delivery Manager and WebOps work at the delivery team level.
+
+##Joiners##
+
+New Developer or WebOps team members will need:
+
+- GitHub Access
+- AWS Access
+- Membership of their delivery teams GitHub team
+- Access to Jira and Confluence
+- Any team specific admin tool access needed in their role.
```

Comment:
Are these outside of the service now ones like o365 / slack etc?

Reply:
Yeah, was thinking things like the LPA Admin Portal (sic)